README.md

```diff
@@ -1,37 +1,65 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
-here [![Travis-CI Build Status](https://travis-ci.org/r-lib/here.svg?branch=master)](https://travis-ci.org/r-lib/here)
-========================================================================================================================
 
-A simple interface to [`rprojroot`](https://github.com/r-lib/rprojroot).
+# here
 
-Motivation
-----------
+<!-- badges: start -->
 
-The logic to find a project root based on certain criteria is packaged in the `rprojroot` package. Many projects might not need the full power, and the complexity that comes with it. Meet `here`:
+[![Travis-CI Build
+Status](https://travis-ci.org/r-lib/here.svg?branch=master)](https://travis-ci.org/r-lib/here)
+[![CRAN
+status](https://www.r-pkg.org/badges/version/here)](https://CRAN.R-project.org/package=here)
+[![Lifecycle:
+stable](https://img.shields.io/badge/lifecycle-stable-brightgreen.svg)](https://www.tidyverse.org/lifecycle/#stable)
+<!-- badges: end -->
 
-``` r
-library(here)
-#> here() starts at /home/travis/build/r-lib/here
-here()
-#> [1] "/home/travis/build/r-lib/here"
-here("DESCRIPTION")
-#> [1] "/home/travis/build/r-lib/here/DESCRIPTION"
-here("R", "here.R")
-#> [1] "/home/travis/build/r-lib/here/R/here.R"
-```
+The goal of the here package is to enable easy file referencing. In
+contrast to using `setwd()`, which is fragile and dependent on the way
+you organize your files, here uses the top-level directory of a project
+to easily build paths to files.
 
-Installation
-------------
+## Installation
 
-Install from CRAN via
+Install the released version of here from CRAN:
 
 ``` r
 install.packages("here")
 ```
 
-Install from GitHub via
+Or install the development version from GitHub with:
 
 ``` r
 devtools::install_github("r-lib/here")
 ```
+
+## Usage
+
+The here package creates paths relative to the top-level directory. The
+package displays the top-level of the current project on load or any
+time you call `here()`:
+
+``` r
+library(here)
+#> here() starts at /Users/sharla/github/tidy-dev-day-2020/here
+here()
+#> [1] "/Users/sharla/github/tidy-dev-day-2020/here"
+```
+
+You can build a path relative to the top-level directory in order to
+read or write a file:
+
+``` r
+here("files", "data", "iris.csv")
+```
+
+``` r
+write.csv(iris, here("files", "data", "iris.csv"))
+```
+
+These relative paths work regardless of where the associated source file
+lives inside your project, like analysis projects with data and reports
+in different
+subdirectories.
+
+![](/Users/sharla/github/tidy-dev-day-2020/here/man/figures/illustration.png)<!-- -->
```

Comment:
Will this work on my machine?

Reply:
🙃 of course it won't! i thought i was being clever to use `here()` here - relative reference would be much better